### ISSUE TITLE
Playing fast is complicated

### ISSUE THREAD

**Issue by tydan** [lukasmonk/lucaschessR#54]
When I play a blitz game (something like 5+2) and I'm in zeitnot, it ends that I misclick because the interface is freezing a few seconds when I release my piece.
I'm on Windows 10 and my config is not too old (i5/gamer graphics card).

**Comment by lukasmonk** [comment]
Are you playing with the tutor activated?

**Comment by tydan** [comment]
No, just the time control 5 minutes 2 seconds, zeitnot 20s and maia1200 as opponent

**Comment by tydan** [comment]
ok I made another test.
It's not happening during zeinot.
You just have to grab a piece and hold it above the square you want to put it after the computer plays.
When it plays, you release the piece and "then" it freezes 1 or 2 seconds.
Sometimes the piece shitfed (for example you try to release on d7, then you release the mouse click the piece appears on g3). Very disturbing.
Can you reproduce ?

**Comment by tydan** [comment]
example of freeze
![LucasR_DNb7LDFhzW](https://user-images.githubusercontent.com/11915916/133942783-64a23094-00e9-4f50-af14-3ecff83ccd86.png)

**Comment by tydan** [comment]
Oh by the way, Lucas Chess best chess program in the world ;)

**Comment by lukasmonk** [comment]
What about to play with these conditions:
![image](https://user-images.githubusercontent.com/6254278/135748221-4f101fdf-3e17-4ffd-b040-87f67b5d39a6.png)

![image](https://user-images.githubusercontent.com/6254278/135748254-3d48bf3b-19b4-40ad-9533-62e883090be1.png)

![image](https://user-images.githubusercontent.com/6254278/135748322-f840f7ff-6969-4b0a-8955-8ed2aa5215a7.png)

**Comment by lukasmonk** [comment]
You could try this configuration:

![image](https://user-images.githubusercontent.com/6254278/135748622-f0cb4910-3ecc-4b47-86cd-4d28004edbb4.png)

![image](https://user-images.githubusercontent.com/6254278/135748653-2a679923-1336-4468-9c47-22b11eeb3faf.png)

**Comment by tydan** [comment]
That was my default configuration, so the problem is here (fixed deep or not).

Looking the configuration menu, one way to attenuate the problem is to set Speed to 9.
Indeed my piece freezes at the moment I release my mouse pressed buttun AND the piece is been moved by computer.
So if the piece moves faster you have less chance to make it freeze. 
But this is a turn around, not a solution :(

**Comment by lukasmonk** [comment]
If the speed has some effect then the field Visual effects is marked (It is speed for visual effects).

The fixed depth in Maia has one effect and that is that the thinking time is real, if not it is a simulation to try to make the behaviour seem more human.

To give a solution I need a context with the problem to replicate it in my PC.
I believe that the problem are the visual effects, but I try to make sure first before spending time on.
There are more reasons that can affect like sounds.

**Comment by tydan** [comment]
Ok this is a little video with the problem.
You have the configuration, no visual effect and same problem when it's on.
On f4, I released the button BEFORE Maia plays (premove) => it's ok
On h4, I released the button AFTER Maia plays  => freeze
The timer is still on my side, I need to click again to make the move accepted.
I hope this will be clear this time ;)

https://user-images.githubusercontent.com/11915916/135903321-d8f3ae37-9b68-4bbf-846d-590978c9701e.mp4
